src/services/PostEventProcessor.ts

```diff
@@ -145,8 +146,8 @@ export class PostEventProcessor {
 			};
 
 		} catch (error: any) {
-			this.logger.error({ message: "Cannot parse event data" });
-			throw new AppError( HttpCodesEnum.BAD_REQUEST, "Cannot parse event data");
+			this.logger.error({ message: "Cannot parse event data", error });
+			throw new AppError(HttpCodesEnum.BAD_REQUEST, "Cannot parse event data");
```

Comment:
might logging the entirety of `error` reveal PII from the message that caused the error?

Reply:
Good point - I've double checked the possible errors that could be thrown in that `try` block, they all come from code that we own and none of them contain PII (they are all just `message` and `statusCode`, where none of the messages include PII)